src/widgets/Posts/Posts.tsx

```
@@ -1,30 +1,39 @@
     variables: { endCursorPostId, searchTerm: debouncedValue },
   })
   const { data: newPostAdded } = usePostAddedSubscription()
 
   const handleSearch = (value: string) => {
+    setAllPosts([])
+    setEndCursorPostId(0)
     setSearchTerm(prevState => value)
   }
 
   useEffect(() => {
     if (posts && posts?.getPosts.items.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)